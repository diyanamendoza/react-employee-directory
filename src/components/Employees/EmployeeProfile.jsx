import styles from '../../App.css'

export default function Profile({employee}) {
    return (
        <div className={styles.card}>
            <div className={styles.cardheader}>
                <p>{employee.name}</p>
                <p>{employee.email}</p>
                <p>{employee.birthday}</p>
            </div>

            <div className={styles.cardbody}>
                <p>{employee.bio}</p>
            </div>

        </div>
    )
}
