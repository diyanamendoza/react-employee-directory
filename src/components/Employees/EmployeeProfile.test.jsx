import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import EmployeeProfile from './EmployeeProfile'

it('should display the provided employee', () => {
    const { container } = render(
        <MemoryRouter>
            <EmployeeProfile employee={{
                name: 'name',
                email: 'email',
                birthday: 'birthday',
                bio: 'bio'
            }} />
        </MemoryRouter>
    )
    
    expect(container).toMatchSnapshot();
   
})