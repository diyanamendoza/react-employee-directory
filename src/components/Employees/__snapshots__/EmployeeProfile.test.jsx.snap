// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the provided employee 1`] = `
<div>
  <div
    class="card"
  >
    <div
      class="cardheader"
    >
      <p>
        name
      </p>
      <p>
        email
      </p>
      <p>
        birthday
      </p>
    </div>
    <div
      class="cardbody"
    >
      <p>
        bio
      </p>
    </div>
  </div>
</div>
`;
